src/VisualStudio/Core/Def/Implementation/RQName/RQNameStrings.cs

```diff
@@ -5,37 +5,37 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.RQName
     internal sealed class RQNameStrings
     {
         internal const string Namespace = "Ns";
-        internal const string Agg = "Agg";
+        internal const string Agg = nameof(Agg);
```

Comment:
These are values defined by a spec; I would change these back because renaming the constants should _not_ change the values.